Jelly/Classes/private/PresentationController.swift

```diff
@@ -45,6 +47,8 @@ final class PresentationController: UIPresentationController, PresentationContro
                 self.setupDimmingView(withAlpha: alpha)
                 animateDimmingView(alpha: alpha)
         }
+        screenshot = presentingViewController.view.snapshotView(afterScreenUpdates: false)!
+        presentingViewController.view!.addSubview(screenshot)
```

Comment:
We should not use force unwraps over here.

Reply:
Will update as well